kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/CreateQuizSection.vue

```diff
@@ -0,0 +1,533 @@
+              @click="toggleItemID(item.id)"
+            >
+              <template
+                #heading="{ title }"
+                :accordionToggle="onAccordionToggle(item.id)"
+              >
+                <KGrid class="question-row">
+                  <KGridItem :layout12="{ span: 6 }">
+                    <div class="left-column-alignment-style">
+                      <DragHandle>
+                        <!-- FIXME: Needs vertically centered -->
+                        <KIcon
+                          class="drag-icon icon-size"
+                          icon="dragVertical"
+                        />
+                      </DragHandle>
+
+                      <div>
+                        <p>
+                          <KCheckbox />
+                        </p>
+                      </div>
+
+                      <div>
+                        <p>
+                          {{ title }}
+                        </p>
+                      </div>
+                    </div>
+                  </KGridItem>
+
+                  <KGridItem :layout12="{ span: 6 }">
+                    <div class="right-alignment-style">
+                      <KIconButton
+                        v-if="isItemExpanded(item.id)"
+                        class="icon-size"
+                        icon="chevronUp"
+                        @click="toggleItemState(item.id)"
+                      />
+
+                      <KIconButton
+                        v-else
+                        class="icon-size"
+                        icon="chevronRight"
+                        @click="toggleItemState(item.id)"
+                      />
+                    </div>
+                  </KGridItem>
+                </KGrid>
+              </template>
+
+              <template
+                v-if="isItemExpanded(item.id)"
+                #content=""
+              >
+                <div class="accordion-detail-container">
+                  <KGrid>
+                    <KGridItem :layout12="{ span: 8 }">
+                      <p>{{ $tr('questionPhrase') }}</p>
+                      <p>{{ $tr('questionSubtitle') }}</p>
+                    </KGridItem>
+
+                    <KGridItem :layout12="{ span: 4 }">
+                      <KIcon class="float-item-left-style" icon="edit" />
+                    </KGridItem>
+                  </KGrid>
+
+                  <div class="choose-question question">
+                    <p class="space-content">
+                      {{ $tr('chooseQuestionLabel') }}
+                    </p>
+                  </div>
+
+                  <div class="question">
+                    <AnswerOption
+                      v-for="answer in item.placeholderAnswers"
+                      :key="answer.id"
+                      :optionId="answer.id"
+                      :isOptionSelected="isOptionSelected"
+                      :isAnswerSelected="isAnswerSelected(answer.id)"
+                    >
```

Comment:
I'm sorry I wasn't clear about this - but the question editing business is not necessary here. We won't be doing any kind of question editing in this pass, so this PR just needs to have the accordion component ready to be fleshed out in a follow-up which will wire up everything in #11088 to here, rendering the questions we get from the DB